interpreter/chapter.txt

```diff
@@ -141,7 +141,7 @@ At this point, we can begin to see how this structure is extensible: we can add
 
 #### Variables
 
-Next let's add variables to our interpreter. Variables require four pieces: an instruction for storing the value of a variable, `STORE_NAME`; an instruction for retrieving it, `LOAD_NAME`; and a mapping from variable names to values. For now, we'll ignore namespaces and scoping, so we can store the variable mapping on the interpreter object itself. Finally, we'll have to make sure that `what_to_execute` has a list of the variable names, in addition to its list of constants.
+Next let's add variables to our interpreter. Variables require an instruction for storing the value of a variable, `STORE_NAME`; an instruction for retrieving it, `LOAD_NAME`; and a mapping from variable names to values. For now, we'll ignore namespaces and scoping, so we can store the variable mapping on the interpreter object itself. Finally, we'll have to make sure that `what_to_execute` has a list of the variable names, in addition to its list of constants.
```

Comment:
As I mentioned in https://github.com/aosabook/500lines/pull/146/, I believe the original version of this paragraph was counting the `what_to_execute` part in the last sentence as piece 4. I don't think we lose anything by removing the "four pieces" reference.